keyboards/akko/5075/config.h

```diff
@@ -0,0 +1,66 @@
+/* Copyright (C) 2023 jonylee@hfd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Use 6 dynamic keymap layers */
+#define DYNAMIC_KEYMAP_LAYER_COUNT 6
+
+/*encoder resolution */
+#define ENCODER_DEFAULT_POS 0x3
+
+/* DIP switch */
+#define DIP_SWITCH_PINS { A9 }
+
+/* LED Indicators */
+#define LED_MAC_OS_PIN      C10
+#define LED_WIN_LOCK_PIN    C11
+
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* SPI Config for spi flash*/
+#define SPI_DRIVER SPIDQ
+#define SPI_SCK_PIN B3
+#define SPI_MOSI_PIN B5
+#define SPI_MISO_PIN B4
+#define SPI_MOSI_PAL_MODE 5
+
+#define EXTERNAL_FLASH_SPI_SLAVE_SELECT_PIN C12
+#define WEAR_LEVELING_BACKING_SIZE (8 * 1024)
+
+/* I2C Config for LED Driver */
+#define DRIVER_COUNT 2
+#define DRIVER_ADDR_1 0b1110100
+#define DRIVER_ADDR_2 0b1110111
```

Comment:
These addresses are invalid for the 3733. What LED driver is this actually using?

Reply:
Except for the addresses of LED driver, everything else is the same as 3733

Reply:
SNLED27351J  @fauxpark

Reply:
I see. It looks like the register layout is quite similar, but there are some changes particularly on the function page, and it does not seem to be a pin-compatible drop-in replacement (3733 only comes in QFN-48 and eTFQP-48).

So although this may work right now, I think a new driver is probably warranted, especially if you have more boards in the pipeline with this chip.

Do these all use the SNLED2730 series?
```
akko/5108
akko/acr87
akko/top40
chosfox/cf81
inland/kb83
inland/mk47
monsgeek/m1
monsgeek/m5
monsgeek/m6
```

Reply:
yes，they all use SNLED2730 @fauxpark

Reply:
Thanks. I've also discovered the CKLED2001 used by the Keychron boards is essentially identical as well, so it would be best to use that driver instead, and I'll work on renaming everything to SNLED in another PR.